src/TinyHelpers/Extensions/StringExtensions.cs

```diff
@@ -21,4 +22,14 @@ public static string ReplaceIgnoreCase(this string input, string pattern, string
 
     public static string? GetValueOrDefault(this string? input, string? defaultValue = default, bool whitespaceAsEmpty = true)
         => whitespaceAsEmpty ? (string.IsNullOrWhiteSpace(input) ? defaultValue : input) : (string.IsNullOrEmpty(input) ? defaultValue : input);
+
+    public static string Repeat(this string str, int times)
+    {
+        var s = new StringBuilder();
+        for (int i = 0; i < times; i++)
+        {
+            s.Append(str);
```

Comment:
What happens if `str` is null or `times` is negative? Perhaps it makes sense to add some parameter checks above.